packages/scaffolding-cli/templates/src/ssr/server/index.ts

```diff
@@ -2,7 +2,7 @@ import * as AI from "applicationinsights"
 import bodyParser from "body-parser"
 import express from "express"
 import next from "next"
-import conf from "../environment-configuration"
+// import conf from "../environment-configuration"
```

Comment:
Are we not using env config anymore?

Reply:
We are not going to use it for most cases.